AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/rootfragment/RootFragmentTest.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (C) 2016-2019 the AndroidAnnotations project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.test.rootfragment;
+
+import static org.fest.assertions.api.Assertions.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+
+@RunWith(RobolectricTestRunner.class)
+public class RootFragmentTest {
+
+	@Test
+	public void injectFragmentWhichInjectedBean() {
+		CustomFragment fragment = new CustomFragment_();
```

Comment:
I would create the `Fragment` properly using `Robolectric`:
```java
CustomFragment fragment = Robolectric.buildFragment(CustomFragment_.class).create().get();
```

This applies to other test methods as well.

Reply:
Yeap, I tried to follow how I saw it in others.. mm.. but if the Fragment is not "added" in any FragmentManager, I don't think it would make that difference using Robolectric for it... or is there any?

Reply:
Robolectric is adding it to a `FragmentManager`, using the method i described above.

Reply:
Ah got it, if so then let's do it like that, since that will bring it closer to the real runtime scenario.